addons/stock/models/stock_location.py

```diff
@@ -259,7 +259,7 @@ def copy(self, default=None):
             default['name'] = _("%s (copy)", self.name)
         return super().copy(default=default)
 
-    def _get_putaway_strategy(self, product, quantity=0, package=None, packaging=None, additional_qty=None):
+    def _get_putaway_strategy(self, product, locations=None, quantity=0, package=None, packaging=None, additional_qty=None):
```

Comment:
Don't put the new param in the middle of the method definition. In case someone is calling this function without named parameter, and only positionnal ones. It's gonna fail